__tests__/AmplifyGeofenceControl.test.ts

```diff
@@ -72,7 +72,7 @@ describe("AmplifyGeofenceControl", () => {
     const control = new AmplifyGeofenceControl({
       geofenceCollectionId: "anyString",
     });
-    control._map = new maplibreMap();
+    control._map = new maplibreMap({ container: '', style: '' });
```

Comment:
@slaymance is this the required arguments you were referring to during standup?

Reply:
Yep, the `MapOptions` type in v2 has those [properties marked as required](https://github.com/maplibre/maplibre-gl-js/blob/1f123972d2e80498e437cf63be24ae33a059888c/src/ui/map.ts#L62-#L107), while in v1 it accepted `MapboxOptions` which [had no such requirements](https://github.com/maplibre/maplibre-gl-js/blob/v1.15.2/src/index.d.ts#L235).

Reply:
Awesome yep this change should be fine as effectively there will be no change to the `createMap` function right?